src/main/resources/templates/clojure-lsp/clientSettings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
All of the languages integrated via these templates are for case-sensitive languages except for HTML (see below).